mindmaps-migration/base/src/main/java/io/mindmaps/migration/base/AbstractMigrator.java

```diff
@@ -34,7 +34,7 @@
 public abstract class AbstractMigrator implements Migrator {
 
     public static final int BATCH_SIZE = 25;
-    public final QueryBuilder queryBuilder = Graql.withoutGraph();
+    public final QueryBuilderImpl queryBuilder = (QueryBuilderImpl) Graql.withoutGraph();
```

Comment:
I'll make a pull req to add this method to the interface. There doesn't seem to be any reason not to.